README.md

```diff
@@ -164,6 +165,45 @@ end
 *NOTE*: This kind of Composite Mapper is static, therefore if you want to have multiple combinations of mappers
  for multiple entries, you'd need to write code a bit differently.
 
+### Structured Text
```

Comment:
What do you think about adding `ALPHA` in here also

Reply:
Sure